src/activity_item_handling.cpp

```diff
@@ -3177,6 +3180,82 @@ static std::optional<tripoint_bub_ms> find_refuel_spot_trap(
     return {};
 }
 
+// Visits an item and all its contents.
```

Comment:
```suggestion
// Visits an item and all its contents.,
```
Resetting test suite